### PR TITLE
Finalize cmdline example migrations and updates

### PR DESCRIPTION
**Pull request by parlough**

### CODE REVIEW

File: examples/misc/test/samples_test.dart [hunk 1]
```diff
@@ -23,16 +23,6 @@ void main() {
   const oneSecond = Duration(seconds: 0);
   final someDate = DateTime(1999);
 
-  test('hello world', () {
```
Comment (kwalrath) [inline examples/misc/test/samples_test.dart:26]: Why remove this?

**Review by kwalrath — APPROVED** [review]
LGTM, but I'm curious about the test. Also... should this PR update the try-dart section to use single quotes for the Hello World example?

File: examples/misc/test/samples_test.dart (hunk 1 above)
Comment (parlough) [reply examples/misc/test/samples_test.dart]: Not sure that I remember any good reasoning. I've added it back!

**Comment by parlough** [comment]
Good point on fixing the try-dart section here, thanks! Luckily the compile resulted in minimal changes.

**Pull request merged**